pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -535,8 +534,8 @@ func (p *provider) Create(ctx context.Context, machine *clusterv1alpha1.Machine,
 
 	// we need this check until this issue is resolved:
 	// https://github.com/kubevirt/containerized-data-importer/issues/895
-	if len(dataVolumeName) > 63 {
-		return nil, fmt.Errorf("dataVolumeName size %v, is bigger than 63 characters", len(dataVolumeName))
+	if len(c.DataVolumeName) > 63 {
+		return nil, fmt.Errorf("dataVolumeName size %v, is bigger than 63 characters", len(c.DataVolumeName))
```

Comment:
I wonder if we really need this check since the name for resources in Kubernetes is already limited to 63 characters. So `machine.name` will never exceed this limit.